docs/DeveloperGuide.adoc

```diff
@@ -345,6 +345,30 @@ image::UndoRedoActivityDiagram.png[width="200"]
 **Cons:** Requires dealing with commands that have already been undone: We must remember to skip these commands. Violates Single Responsibility Principle and Separation of Concerns as `HistoryManager` now needs to do two different things. +
 // end::undoredo[]
 
+// tag::sortcommand[]
+=== Sort mechanism
+
+The sort mechanism is facilitated by an `Event`. When a sort command is executed, the sort command will conduct a sort on the current `list` in the address book based on the full name in alphabetical order.
+
+The following sequence diagram shows how the sort operation works:
+
+image:: SortCommandSequenceDiagram.png[width="800"]
```

Comment:
The image is not shown, remove the space between image:: and SortCommandSequenceDiagram.png[width="800"]